src/Prompt.php

```diff
@@ -249,7 +249,7 @@ public static function terminal(): Terminal
     /**
      * Set the custom validation callback.
      */
-    public static function validateUsing(Closure $callback): void
+    public static function validateUsing(?Closure $callback): void
```

Comment:
Note that `$validateUsing` property [already allows](https://github.com/laravel/prompts/pull/186/files#diff-669bd09458cf69d12a4c7ddae353f28eb26cf6bb4ee3d94fd5fa4aabb65b2450R77) to be set to `NULL`.